daemon/src/monitor.rs

```diff
@@ -746,6 +877,15 @@ where
             collaborative_settlement.order_id,
         );
     }
+
+    async fn handle_try_broadcast_transaction(
+        &mut self,
+        msg: TryBroadcastTransaction,
+        _ctx: &mut xtra::Context<Self>,
```

Comment:
`xtra-productivity` can handle functions that don't need `xtra::Context`! Just omit it!